react/features/base/dialog/reducer.js

```diff
@@ -0,0 +1,33 @@
+import { ReducerRegistry, setStateProperties } from '../redux';
```

Comment:
Seems fine to me. 2 imports, one is ours '../redux', the other one is separate by newline as it is from the same feature './actionTypes'.